adage/analyze/api.py

```diff
@@ -5,7 +5,7 @@
 from organisms.api import OrganismResource
 from genes.api import GeneResource
 from tastypie import fields, http
-from tastypie.resources import Resource, ModelResource
+from tastypie.resources import Resource, ModelResource, convert_post_to_VERB
```

Comment:
I've added several tests now, aimed at exercising the ExpressionValue API the way we have tested our other APIs.

PLEASE NOTE: although there is a test included to ensure that we can retrieve data using the POST API method that would normally fail a GET, this test is somewhat non-informative because the failure actually takes place in the nginx reverse-proxy layer, which cannot be tested without a complete E2E test infrastructure.